app/services/LocationService.js

```diff
@@ -296,40 +303,77 @@ export default class LocationServices {
 
       if (status !== BackgroundGeolocation.AUTHORIZED) {
         // we need to set delay or otherwise alert may not be shown
-        // setTimeout(
-        //   () =>
-        //     Alert.alert(
-        //       languages.t('label.require_location_information_title'),
-        //       languages.t('label.require_location_information_message'),
-        //       [
-        //         {
-        //           text: languages.t('label.yes'),
-        //           onPress: () => BackgroundGeolocation.showAppSettings(),
-        //         },
-        //         {
-        //           text: languages.t('label.no'),
-        //           onPress: () => console.log('No Pressed'),
-        //           style: 'cancel',
-        //         },
-        //       ],
-        //     ),
-        //   1000,
-        // );
+        setTimeout(
+          () =>
+            Alert.alert(
+              languages.t('label.require_location_information_title'),
+              languages.t('label.require_location_information_message'),
+              [
+                {
+                  text: languages.t('label.yes'),
+                  onPress: () => BackgroundGeolocation.showAppSettings(),
+                },
+                {
+                  text: languages.t('label.no'),
+                  onPress: () => {
+                    console.log('No Pressed'),
+                      (isBackgroundGeolocationConfigured = false);
+                  },
+                  style: 'cancel',
+                },
+              ],
+            ),
+          1000,
+        );
       } else {
         BackgroundGeolocation.start(); //triggers start on start event
 
         BackgroundGeolocation.checkStatus(({ locationServicesEnabled }) => {
           if (!locationServicesEnabled) {
-            PushNotification.localNotification({
-              id: LOCATION_DISABLED_NOTIFICATION,
-              title: languages.t('label.location_disabled_title'),
-              message: languages.t('label.location_disabled_message'),
-            });
-          } else {
-            PushNotification.cancelLocalNotifications({
-              id: LOCATION_DISABLED_NOTIFICATION,
-            });
+            //we need to set delay or otherwise alert may not be shown
+            setTimeout(
```

Comment:
could we pull this big alert body into its own method here? something along the lines of `alertRequiredLocationServices()`?